internal/pkg/template/templates/workloads/partials/cf/rollback-alarms.yml

```
@@ -0,0 +1,45 @@
+{{- if .DeploymentConfiguration.CPUUtilization}}
+CPURollbackAlarm:
+  Metadata:
+    'aws:copilot:description': "A CloudWatch alarm associated with CPU utilization for deployment rollbacks"
+  Type: AWS::CloudWatch::Alarm
+  DependsOn: Service
+  Properties:
+    AlarmDescription: "Roll back ECS service if CPU utilization is greater or equal to {{.DeploymentConfiguration.CPUUtilization}}% twice in 3 minutes."
+    AlarmName: !Sub ${AWS::StackName}-CPUAlarm
```

I just want to be as specific as possible since the customer can add its own custom alarms and we don't want to clash with names: (same for the others)
```suggestion
    AlarmName: !Sub ${AWS::StackName}-CopilotRollbackCPUAlarm
```

---

Also I wonder if we should do some truncating here given that alarms have a limit of 255 chars 🤔 

Something like this (I'm making this up, feel free to do something else):
```suggestion
    AlarmName: {{.DeploymentConfiguration.AlarmName .App .Env .Name "CopilotRollbackCPUAlarm"}}
```

```go
func (cfg DeploymentConfig) AlarmName(app, env, name, suffix string) string {
   maxChunk := (255 - len(suffix) - 3 /* number of dashes */) / 3
   return fmt.Sprintf("%s-%s-%s-%s", app[:maxChunk], env[:maxChunk], name[:maxChunk], suffix)
}
```

---

Yeah, good call!
Is it too late to do something similar [here](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/template/templates/environment/partials/ar-vpc-connector.yml#L7) and [here](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/template/templates/environment/partials/ar-vpc-connector.yml#L11) so it's not the end that gets lopped off?

---

If the updates don't require replacement it'd be great!